.github/actions/add_conan_provider/action.yml

```diff
@@ -6,6 +6,4 @@ runs:
       - name: Get dependency provider
         shell: bash
         run: |
-          sudo apt-get install -y wget
-          
-          wget https://github.com/conan-io/cmake-conan/raw/develop2/conan_provider.cmake -O conan_provider.cmake
+          curl -L https://github.com/conan-io/cmake-conan/raw/develop2/conan_provider.cmake -o conan_provider.cmake
```

Comment:
It would probably also make sense to pin this to `https://raw.githubusercontent.com/conan-io/cmake-conan/b0e4d1ec08edb35ef31033938567d621f6643c17/conan_provider.cmake`